src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests2.cs

```diff
@@ -424,7 +424,7 @@ public static void Main()
         var r2 = b switch { (true ? true : true) => true, false => false };
     }
 }";
-            // PROTOTYPE(patterns2): This is admittedly poor syntax error recovery (for the line declaring r2),
+            // This is admittedly poor syntax error recovery (for the line declaring r2),
```

Comment:
Are we admitting that there's no way to make this better? Otherwise this may be worth an issue.

Reply:
I'd rather wait until a report that someone is actually confused by the behavior.

---
In reply to: [194896101](https://github.com/dotnet/roslyn/pull/27753#discussion_r194896101) [](ancestors = 194896101)